apps/codeforafrica/src/components/ArticleHeader/ArticleHeader.js

```diff
@@ -50,7 +58,12 @@ const ArticleHeader = React.forwardRef(function ArticleHeader(props, ref) {
       {tags?.length > 0 ? (
         <ChoiceChipGroup color="default" sx={{ mt: { xs: 2.5, md: 5 } }}>
           {tags.map((tag) => (
-            <ChoiceChip label={tag} value={tag} key={tag} />
+            <ChoiceChip
+              label={tag}
+              value={tag}
+              key={tag}
+              onClick={handleChangeTag}
```

Comment:
`import { Link } from "@commons-ui/next"` at the top of file and:
```suggestion
              component={Link}
              href={`/${primaryTag.slug}?tag=${tag}`
```

No need to deal with router/click handlers.